docs/configuration/production.rst

```diff
@@ -19,18 +19,31 @@ Install and setup PostgreSQL
 
 (*requires PostgreSQL 9.4 or higher*).
 
-*Kinto* dependencies do not include *PostgreSQL* tooling and drivers by default.
+*Kinto* dependencies do not include *PostgreSQL* tooling and drivers by default, which should be installed and configured before proceeding to the next steps. More information is available at the `PostgreSQL Documentation <http://www.postgresql.org/docs>`_.
 
 
 PostgreSQL client
 -----------------
 
+Before installing the Python libraries for Postgres we need to first install the header files for the PostgreSQL database backend.
+
 On Debian / Ubuntu based systems::
 
     $ sudo apt-get install libpq-dev
 
+On RedHat / Fedora / Mint based systems::
+
+    $ yum install postgresql-devel
+
 On Mac OS X, `install a server or use port <http://superuser.com/questions/296873/install-libpq-dev-on-mac-os>`_.
 
+Install PostgreSQL Python Dependencies
+--------------------------------------
+
+Kinto PostgreSQL backend rely on several Python toolkits, which can be installed with a single *pip* command ::
+
+    $ sudo pip install -Ue kinto[postgresql]
```

Comment:
Do not mention `sudo` here. Even though it may work, it installs the Python libraries globally on the system, which is not recomended.

Also, I think `-U` is enough.